lib/ui/window.dart

```diff
@@ -289,10 +289,10 @@ class FlutterView {
   ///   scheduling of frames.
   /// * [RendererBinding], the Flutter framework class which manages layout and
   ///   painting.
-  void render(Scene scene) => _render(scene);
+  void render(Scene scene) => _render(viewId == 0 ? 0 : viewId.hashCode, scene);
```

Comment:
Why `viewId.hashCode`? Should this be `viewId as int`?
cene);

Reply:
Good point, it was a panic move because I didn't know how to convert Object to int...